### PR TITLE
Upgrade to @material-ui/core

### LINKED ISSUE #2: Update to @material-ui/core
Hello

I am using material-ui-toggle-icon in a project with material-ui v1.0.0-rc.0, and got this error:
```
ERROR in ./node_modules/material-ui-toggle-icon/lib/ToggleIcon.js
Module not found: Error: Can't resolve 'material-ui/styles'
```

### PR DESCRIPTION
**Pull request by timothyvanderaerden**
This should fix #2, I hope I did everything correctly.

### CODE REVIEW

**Comment by leMaik** [comment]
Awesome, thank you very much! :tada:

File: src/ToggleIcon.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ const styles = {
     width: 24,
     height: 24,
     position: 'relative',
-    display: 'inline-block'
+    display: 'inline-block',
+    transition: 'clip-path 550ms cubic-bezier(0.4, 0.0, 0.2, 1), -webkit-clip-path 550ms cubic-bezier(0.4, 0.0, 0.2, 1)'
```
Comment (leMaik) [inline src/ToggleIcon.js]: Adding this broke the animation, which is the sole purpose of this component. :smile:

**Comment by leMaik** [comment]
I released the updated version. :tada:

File: src/ToggleIcon.js (hunk 1 above)
Comment (timothyvanderaerden) [reply src/ToggleIcon.js]: Oops, sorry! Not sure how that ended up there, weird that I didn't noticed it. I'm sure I tested it a couple of times :disappointed:

**Pull request merged**